tests/unit/net/fetch.rs

```diff
@@ -203,6 +204,7 @@ fn test_cors_preflight_fetch() {
         if request.method == Method::Options && state.clone().fetch_add(1, Ordering::SeqCst) == 0 {
             assert!(request.headers.has::<AccessControlRequestMethod>());
             assert!(request.headers.has::<AccessControlRequestHeaders>());
+            assert!(request.headers.has::<HyperReferer>());
```

Comment:
The important piece here is that we want to verify that the header does not contain `a.html`, since that should have been stripped out when we used the `Origin` referrer policy.